test/test_transforms_tensor.py

```diff
@@ -714,6 +714,7 @@ def test_random_apply(device):
 @pytest.mark.parametrize('channels', [1, 3])
 def test_gaussian_blur(device, channels, meth_kwargs):
     tol = 1.0 + 1e-10
+    torch.manual_seed(12)
```

Comment:
I had to add this to prevent the test from failing (see https://app.circleci.com/pipelines/github/pytorch/vision/10830/workflows/946973f0-bad1-48ed-aaf3-f3720ab5f56a/jobs/828462).

If anything, this shows that the PR is working as expected and that this test is a bit flaky, and sensible to the `_create_data()` call.

To confirm I parametrized it over 100 random seeds, and I got 8 failures over 1000+ test instances. Each time, just 1 pixel was off. Considering the low failure rate, I think it's fine to keep the manual seeding here.